src/process.rs

```diff
@@ -99,14 +99,16 @@ pub async fn process(
 
     let poll_data = poll_data_main.clone();
 
-    tokio::spawn(async move {
-        let mut proc = Exec::cmd(process)
-            .arg(args)
-            .env_extend(&env_vars)
-            .popen()
-            .expect("Failed to start process");
+    let mut proc = Exec::cmd(process)
+        .arg(args)
+        .env_extend(&env_vars)
+        .stdout(Redirection::Pipe)
+        .stderr(Redirection::Pipe)
```

Comment:
Necessary? We set comms up

Reply:
Yes it is, otherwise it inherits stdio from the parent process (i spent a stupid amount of time looking for a debug print that didn't exist)